platform-tests/src/test/java/org/junit/platform/engine/support/hierarchical/VirtualParallelExecutionIntegrationTests.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.engine.support.hierarchical;
+
+import static org.junit.jupiter.api.condition.JRE.JAVA_19;
+
+import org.junit.jupiter.api.condition.EnabledOnJre;
+
+@EnabledOnJre(value = JAVA_19, disabledReason = "Use Java 19 preview features")
+public class VirtualParallelExecutionIntegrationTests extends ParallelExecutionIntegrationTests {
```

Comment:
In order to run this test class, we need to change the `JavaLauncher` on the `Test` task to use JDK 21.

Reply:
Fixed via https://github.com/junit-team/junit5/pull/3443/commits/fb9f2b7d9d32b8c5e8d2dbfe7a1c3238c97e361d